sui-execution/latest/sui-adapter/src/execution_engine.rs

```diff
@@ -1003,13 +1015,21 @@ mod checked {
 
     fn setup_bridge_create(
         mut builder: ProgrammableTransactionBuilder,
+        chain_id: ChainIdentifier,
     ) -> ProgrammableTransactionBuilder {
         let bridge_uid = builder
             .input(CallArg::Pure(UID::new(SUI_BRIDGE_OBJECT_ID).to_bcs_bytes()))
-            .unwrap();
-        // Set chain id to sui devnet
-        // TODO: set this according to ChainIdentifier
-        let bridge_chain_id = builder.pure(2u8).unwrap();
+            .expect("Unable to create Bridge object UID!");
+
+        let bridge_chain_id = if chain_id == get_mainnet_chain_identifier() {
+            0u8
+        } else if chain_id == get_testnet_chain_identifier() {
+            1u8
+        } else {
+            2u8
+        };
```

Comment:
we do want to distinguish devnet from anything else: https://www.notion.so/mystenlabs/Native-Bridge-V1-4ed9dd1877d042848f72148302062bf4?pvs=4#7acae2bb98cb4eb0854ed5fca7c1fe9f

thoughts?

Reply:
I am not sure if we can do this using `ChainIdentifier` the only "env" value we have are mainnet/testnet/unknown